composer/utils/checkpoint.py

```diff
@@ -67,6 +68,14 @@ def _get_write_mode(name: str) -> str:
     raise ValueError(f'{name} does not end with a valid tarfile extension.')
 
 
+def _parse_load_path(load_path: str):
+    parse_result = urlparse(load_path)
+    if parse_result.scheme == 'wandb':
+        return ''.join([parse_result.netloc, parse_result.path]).lstrip('/').replace('/', '.')
```

Comment:
why is this joined with the empty string? I would expect it to be joined with a /. And also why are / replaced with .? I guess I'm missing something about what a load path is expected to be for wandb.